mora/service/common.py

```diff
@@ -89,21 +89,24 @@ def inactivate_old_interval(old_from: str, old_to: str, new_from: str,
     return payload
 
 
-def set_object_value(obj: dict, path: tuple, vals: List[dict],
+def set_object_value(obj: dict, path: tuple, val: Union[dict, List[dict]],
                      overwrite: bool = False):
     path_list = list(path)
     obj_copy = copy.deepcopy(obj)
 
+    if val is not list:
```

Comment:
This can't possibly work — don't you mean:

```python
if not isinstance(val, list):
```

…but that suggests you're probably not hitting this case in the tests, so you should either adapt them or drop this clause.